theme/src/components/ImgCard.tsx

```diff
@@ -3,7 +3,7 @@
  */
 
 import React from 'react'
-import { Link } from 'gatsby'
+import { default as Link } from '../ContentLink'
```

Comment:
Hi @MattReimer, I've noticed the path to `ContentLink` in the `ImgCard` component is incorrect. I'll make the necessary updates and follow up through a PR.